containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzJettyContainer.java

```diff
@@ -178,6 +179,46 @@ public void addServletHandlers(String serverHostName) {
             rewriteHandler.addRule(disableKeepAliveRule);
         }
         
+        // Security: Respond the "Expect-CT: max-age=31536000, report-uri=..." header
+
+        HeaderPatternRule expectCtRule = new HeaderPatternRule();
+        expectCtRule.setPattern("/*");
+        expectCtRule.setName("Expect-CT");
+        expectCtRule.setValue("max-age=31536000, report-uri=\"http://csp.yahoo.com/beacon/csp?src=yahoocom-expect-ct-report-only\"");
+        rewriteHandler.addRule(expectCtRule);
+
+        // Security: Respond the "Strict-Transport-Security: max-age=31536000" header
+
+        HeaderPatternRule hstsRule = new HeaderPatternRule();
+        hstsRule.setPattern("/*");
+        hstsRule.setName(HttpHeader.STRICT_TRANSPORT_SECURITY.asString());
+        hstsRule.setValue("max-age=31536000");
+        rewriteHandler.addRule(hstsRule);
+
+        // Security: Respond the "X-Content-Type-Options: nosniff" header (see https://docs.google.com/document/d/1ovG_qSqpbS1q2OpEX1Jo08w8dVdi4un1JT7CLz5unX4/edit)
```

Comment:
no links to internal google docs

Reply:
With the new approach (JSON) - this is no longer relevant

Reply:
The links can be added into the zts/zms configuration